test/hotspot/gtest/utilities/test_ostream.cpp

```diff
@@ -121,6 +122,34 @@ TEST_VM(ostream, bufferedStream_dynamic_small) {
   }
 }
 
+static void provoke_snprintf_error() {
+  const wchar_t w[] = { (wchar_t)-1, 0 };
+  char buf[20];
+  ::memset(buf, 'X', sizeof(buf));
+  int n = ::snprintf(buf + 1, sizeof(buf) - 2, "HALLO %ls", w);
+  if (n == -1) { // yes, we get an error. Retry using stringStream.
```

Comment:
Can it happen that we don't get an error, here? If so, the ASSERT below still expects one, so, the test will fail. Correct?

Reply:
Good catch. I added the death test afterwards. I'll fix it.